serving/docker/Dockerfile

```diff
@@ -1,15 +1,24 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 ARG djl_version=0.17.0
+ARG build_type
 
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
     fakeroot \
     openjdk-11-jdk-headless \
     curl
 
-RUN curl -O https://publish.djl.ai/djl-serving/djl-serving_${djl_version}-1_all.deb && \
-    dpkg -i djl-serving_${djl_version}-1_all.deb && \
-    rm djl-serving_${djl_version}-1_all.deb
+COPY *.deb .
+RUN dpkg -i *.deb && rm *.deb
+
+#RUN if [ "$build_type" = "nightly" ] ; then  \
+#      cp *.deb . && \
```

Comment:
If condition does not seem to work. Debugging.

Reply:
You can checkout my PR, we can migrate this logic out of the docker, just assume there is a file. Or you can use Docker multi-stage build to separate

Reply:
Just checked. Shell script seems good Since we need both nightly and release docker images, we can pass an arg to the shell script to see whether to use the local deb file or the published deb. We can reuse the same shell for all dockerfiles.

Will remove my logic of uploading deb to actions' artifacts server.